kuma/javascript/src/search-results-page.jsx

```diff
@@ -109,18 +218,28 @@ export class SearchRoute extends Route<SearchRouteParams, SearchResults> {
     match(url: string): ?SearchRouteParams {
         let parsed = new URL(url, BASEURL);
         let path = parsed.pathname;
-        let q = parsed.searchParams.get('q');
+        let query = parsed.searchParams.get('q');
+        let page = parseInt(parsed.searchParams.get('page') || '1');
```

Comment:
I asked [David](https://www.amazon.com/dp/0596805527), who wrote must of this code in the first place, about that and like the big dummy I am I can't remember why, but he convinced me it's not necessary. None of our `eslint` tooling is bothered by it.

Reply:
Fair enough, just looking at [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Octal_interpretations_with_no_radix) ES5 says that parseInt no longer allows ’0...’ Strings to be treated as octal values so that's probably safe.

Reply:
We're not using ES5. We're using ES6, right? Which transpiles to ES5. Perhaps when it does so, it injects the optional "10" radix thing.

Reply:
Right as long as the browser supports ES5 proper then the behavior for parseint has changed and this would be fine.

Reply:
I don't think babel (or at least the plugins we're using) does any non-syntactical transformations, so it shouldn't touch `parseInt`. But I also don't think we need-need the radix here, as the `page` shouldn't start with `0` or `0x`, so a radix of 10 should be assumed indeed.